deploy/contents/k8s/cp-vm-monitor/cp-vm-monitor-dpl.yaml

```diff
@@ -56,6 +56,9 @@ spec:
             - mountPath: /opt/common/pki
               name: common-pki
               readOnly: true
+            - mountPath: /opt/cp-core-fs
```

Comment:
Let's change path not to overlap with other mounts, e.g. `/fs/core-fs`

Reply:
Moved default mount-point to `/fs`